packages/files-ui/cypress/fixtures/linkData.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable max-len */
+export const viewOnlyShareLink = "http://localhost:3000/link-sharing/read/eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0eXBlIjoibGlua19zaGFyaW5nIiwicGVybWlzc2lvbiI6InJlYWQiLCJpYXQiOjE2Mzc4NzA1NzEsImJ1Y2tldF9pZCI6ImM1ZTA0ZTczLTVkYzUtNDYwOC05ODYyLWE1M2YzY2EzYmM4OSIsIm5vbmNlX2lkIjoiMzZmYjkxNzYtZDA4NS00Nzc5LTg0NTUtNTM0YjcxMzQzZjdkIn0.QE0zwQrY1kFAc_U-FsrJTcWLPMicK5Nq91RdD0szFDmg63R5YRTyBq25xJcOWlIqkAfm_WuwctzWSW5VrgYrJw#SXqdF0tablsTcN%2FaOvAFY9hptaIc4oIH935OBOp9Cg0%3D"
```

Comment:
To be a tiny bit safer, we could use the `localHost` from the `fixtures/loginData` and have:
```suggestion
export const viewOnlyShareLink = `${localHost}/link-sharing/read/eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0eXBlIjoibGlua19zaGFyaW5nIiwicGVybWlzc2lvbiI6InJlYWQiLCJpYXQiOjE2Mzc4NzA1NzEsImJ1Y2tldF9pZCI6ImM1ZTA0ZTczLTVkYzUtNDYwOC05ODYyLWE1M2YzY2EzYmM4OSIsIm5vbmNlX2lkIjoiMzZmYjkxNzYtZDA4NS00Nzc5LTg0NTUtNTM0YjcxMzQzZjdkIn0.QE0zwQrY1kFAc_U-FsrJTcWLPMicK5Nq91RdD0szFDmg63R5YRTyBq25xJcOWlIqkAfm_WuwctzWSW5VrgYrJw#SXqdF0tablsTcN%2FaOvAFY9hptaIc4oIH935OBOp9Cg0%3D`
```

In case we change this in the future..

Reply:
Good idea, thank you

Reply:
I think you forgot to implement this suggestion :)

Reply:
Oops! Thanks

Reply:
Updated now :)